project_name/app.py

```diff
@@ -39,6 +42,14 @@ def read(*paths, **kwargs):
     },
 )
 
+if settings.server and settings.server.get("cors_origins", None):
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=settings.server.cors_origins,
+        allow_credentials=True,
+        allow_methods=["*"],
+        allow_headers=["*"],
```

Comment:
Letstake the other args from settings?

```suggestion
        allow_credentials=settings.get("server.cors_allow_credentials", True),
        allow_methods=settings.get("server.cors_allow_methods",  ["*"]),
        allow_headers=settings.get("server.cors_allow_headers",  ["*"]),
```